plutus-core/plutus-ir/src/PlutusIR/Core/Instance/Pretty/Readable.hs

```diff
@@ -0,0 +1,181 @@
+{-# LANGUAGE ConstraintKinds       #-}
+{-# LANGUAGE LambdaCase            #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE TypeApplications      #-}
+{-# LANGUAGE TypeOperators         #-}
+{-# LANGUAGE UndecidableInstances  #-}
+{-# LANGUAGE ViewPatterns          #-}
+{-# OPTIONS_GHC -Wno-orphans       #-}
+{-# OPTIONS_GHC -Wno-name-shadowing #-}
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
+module PlutusIR.Core.Instance.Pretty.Readable
+  ( prettyPirReadable
+  ) where
+
+import PlutusCore.Default.Universe
+import PlutusCore.Pretty.ConfigName
+import PlutusCore.Pretty.PrettyConst
+import PlutusCore.Pretty.Readable
+import PlutusIR.Core.Type
+import PlutusPrelude
+import Prettyprinter
+import Prettyprinter.Custom
+
+type PrettyPir = PrettyBy (PrettyConfigReadable PrettyConfigName)
+
+-- | Pretty-print something with the Pir prettyprinter settings.
+prettyPirReadable :: PrettyPir a => a -> Doc ann
+prettyPirReadable = prettyBy prettyConfigReadable
+  -- Note, for now this is just a simple default setting. We may want to add more complicated
+  -- settings in the future.
+  where prettyConfigReadable = botPrettyConfigReadable defPrettyConfigName ShowKindsYes
```

Comment:
`ShowKindsNo` here.

Reply:
I propose a third `ShowComplexKinds` option that we would use here. I think having no kinds makes complex generated programs (which I care a lot about) almost unreadable.

Reply:
`instance Default ShowKinds` if you like.

Reply:
I've now made this change and I think it also makes the pretty-printed plc programs a bit more readable.